package-lock.json

```diff
@@ -4833,25 +4833,24 @@
       "dependencies": {
         "abbrev": {
           "version": "1.1.1",
-          "resolved": "https://registry.npmjs.org/abbrev/-/abbrev-1.1.1.tgz",
```

Comment:
Weird that `package-lock` changed for you without a change to `package.json`. When I `npm i` on this branch it updates this file. We should figure out what the differences between our dev environments is.